docs/vcs.rst

```diff
@@ -228,52 +220,50 @@ GitHub
 
 .. versionadded:: 2.3
 
-This just adds a thin layer on top of :ref:`vcs-git` to allow push translation
-changes as pull requests instead of pushing directory to the repository.
-It currently uses the `hub`_ tool to do the integration.
+This adds a thin layer atop :ref:`vcs-git` using the `hub`_ tool to allow pushing
+translation changes as pull requests, instead of
+pushing the directory to the repository.
```

Comment:
the whole
```suggestion
pushing the whole directory to the repository.
```

Reply:
It should actually be "pushing directly to the repository".